java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/XyShapeProperty.java

```diff
@@ -0,0 +1,206 @@
+package org.opensearch.client.opensearch._types.mapping;
```

Comment:
Please add license header

Reply:
Done

Reply:
My apologies, I should have added the header (since this is a new file, it has no attribution to Elasticsearch by OpenSearch only), we need only:

```

/*
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 */


```